### DISCUSSION TITLE
Units without values

### DISCUSSION THREAD

**Discussion by rick-palmsens** [angularsen/UnitsNet#957] [Q&A]
For our project we need to store unit information without actually storing a value.
Is this library capable of doing such things, or does there always have to be a value?

For example, I want to store the unit MicroAmpere.
I've been looking at QuantityInfo and UnitInfo, but I'm not entirely sure how to create something like that.

**Comment by tmilnthorp** [comment]
```UnitsNet.Units.ElectricCurrentUnit.Microampere``` is an enum. Could you just store a ```List<System.Enum>```?

**Reply by rick-palmsens** [reply]
I'd like more type safety than just `System.Enum` 😅
How would I even turn it into something that knows that the SI unit is ampere and that the prefix is micro?

**Reply by tmilnthorp** [reply]
Unfortunately you don't have much choice 👎 

A ```List<(UnitsNet.QuantityInfo, Enum)>``` would certainly be better:

```csharp
var units = new List<(UnitsNet.QuantityInfo, Enum)>();
units.Add( (UnitsNet.ElectricCurrent.Info, UnitsNet.Units.ElectricCurrentUnit.Microampere) );

var (info, unit) = units.First();

var inBase = UnitsNet.Quantity.FromQuantityInfo( info, 1.0 );
var inOriginalUnits = inBase.As( unit );
```
